### ISSUE TITLE
Error when Building on Windows 10

### ISSUE THREAD

**Issue by pedrotrschneider** [bevyengine/bevy#1039]
**Bevy version**

0.3.0

**Operating system & version**

Windows 10

**What you did**

Tried to build an empty cargo project (`cargo build`) with `bevy = "0.3"` as a dependency on the `Cargo.toml` file, as described in the getting started of the bevy book.

**What you expected to happen**

Expected the crates and dependencies to be built properly with no errors.

**What actually happened**

Console threw an error:
``` bash
error: failed to run custom build command for `bevy-glsl-to-spirv v0.2.1`

Caused by:
  process didn't exit successfully: `C:\Users\Estudio\Documents\Pedro\Coding\Rust\bevy\bevy_test\target\debug\build\bevy-glsl-to-spirv-16a5f9db89d32536\build-script-build` (exit code: 101)
  --- stderr
  thread 'main' panicked at 'Missing target support x86_64-pc-windows-gnu', C:\Users\Estudio\.cargo\registry\src\github.com-1ecc6299db9ec823\bevy-glsl-to-spirv-0.2.1\build/build.rs:24:9
```
**Additional information**

Issue #939 also describes a problem building on windows 10, but the cause seems to be different.
I do have Visual Studio 2019 Build Tools installed.

**Comment by memoryruins** [comment]
> Missing target support x86_64-pc-windows-gnu

It will also require the msvc rust toolchain. If `rustup show` shows that msvc is currently the default, it is possible there is another rust installed and is on your path, similar to https://github.com/bevyengine/bevy/issues/797 . If it doesn't show msvc as your default, then `rustup default stable-x86_64-pc-windows-msvc` should be all that is needed.

**Comment by pedrotrschneider** [comment]
> It will also require the msvc rust toolchain. If `rustup show` shows that msvc is currently the default, it is possible there is another rust installed and is on your path, similar to #797 . If it doesn't show msvc as your default, then `rustup default stable-x86_64-pc-windows-msvc` should be all that is needed.

When using `rustup default stable-x86_64-pc-windows-msvc`, I recieve the following output:
``` bash
info: using existing install for 'stable-x86_64-pc-windows-msvc'
info: default toolchain set to 'stable-x86_64-pc-windows-msvc'

  stable-x86_64-pc-windows-msvc unchanged - rustc 1.48.0 (7eac88abb 2020-11-16)
```
No errors, but if I try `cargo build` now I get another error, as if I didn't have cargo:
``` bash
info: syncing channel updates for 'stable-x86_64-pc-windows-msvc'
info: latest update on 2020-11-19, rust version 1.48.0 (7eac88abb 2020-11-16)
info: syncing channel updates for 'stable-x86_64-pc-windows-msvc'
info: latest update on 2020-11-19, rust version 1.48.0 (7eac88abb 2020-11-16)
info: syncing channel updates for 'stable-x86_64-pc-windows-msvc'
info: syncing channel updates for 'stable-x86_64-pc-windows-msvc'
info: latest update on 2020-11-19, rust version 1.48.0 (7eac88abb 2020-11-16)
info: syncing channel updates for 'stable-x86_64-pc-windows-msvc'
info: syncing channel updates for 'stable-x86_64-pc-windows-msvc'
info: latest update on 2020-11-19, rust version 1.48.0 (7eac88abb 2020-11-16)
error: 'cargo.exe' is not installed for the toolchain 'stable-x86_64-pc-windows-msvc'
To install, run `rustup component add cargo`
```
If I run `rustup default stable-x86_64-pc-windows-gnu` to go back to the previous configuration I had, `cargo build` works as described in the original post.
`rustup component add cargo` gives this output:
``` bash
info: latest update on 2020-11-19, rust version 1.48.0 (7eac88abb 2020-11-16)
error: toolchain 'stable-x86_64-pc-windows-msvc' does not support components
```
Did I do any of the steps wrong?

**Comment by pedrotrschneider** [comment]
Just in case someone else finds the same error: what I did was uninstall and reinstall `stable-x86_64-pc-windows-msvc` by running:
`rustup toolchain uninstall stable`
followed by:
`rustup toolchain install stable `
And this alone fixed my problem. I found this solution on an Issue in the rustup repo, and they also said that you might need to reinstall vs build tools 2019 as well. I didn't have to, but keep this in mind.
